### DISCUSSION TITLE
[Resolved] Getting error when promoting a working build to production

### DISCUSSION THREAD

**Discussion by morganpage** [vercel/community#3765] [Help] [answered]
### Summary

request to https://api-sfo1.vercel.com/v2/now/deployments/dpl_3a5etSebiLxcePNJRcRXZ1DHNkpx/builds/bld_99ehhzbbj/ready?ownerId=he5c44er4CFaI8iNRV25bOVc&traceCarrier=%7B%22ot-baggage-clihookAt%22%3A%221692963116133%22%2C%22x-datadog-trace-id%22%3A%228097778659775227960%22%2C%22x-datadog-parent-id%22%3A%225331874497692033303%22%2C%22x-datadog-sampling-priority%22%3A%222%22%2C%22x-datadog-tags%22%3A%22_dd.p.dm%3D-3%22%2C%22traceparent%22%3A%2200-0000000000000000706116c985a42838-49fe9f9b2de5e117-01%22%2C%22tracestate%22%3A%22dd%3Dt.dm%3A-3%3Bs%3A2%22%7D&traceCarrier=%7B%22ot-baggage-clihookAt%22%3A%221692963116133%22%2C%22x-datadog-trace-id%22%3A%228097778659775227960%22%2C%22x-datadog-parent-id%22%3A%229195191665509058946%22%2C%22x-datadog-sampling-priority%22%3A%222%22%2C%22x-datadog-tags%22%3A%22_dd.p.dm%3D-3%22%2C%22traceparent%22%3A%2200-0000000000000000706116c985a42838-7f9be21a004cc982-01%22%2C%22tracestate%22%3A%22dd%3Dt.dm%3A-3%3Bs%3A2%22%7D&traceCarrier=%7B%22ot-baggage-clihookAt%22%3A%221692963116133%22%

### Example

https://vercel.com/morganjpage/rogues-build/9VZpPwD4uGEMj52LJvs37HGeTdBV

### Steps to Reproduce

Promoting a deployment to production

**Comment by SamiHK** [comment]
Facing same issue on https://vercel.com/fynder/fyi-chat/A455M37eytiJNGAMNnh43CzFQ6oE

**Comment by swarnava** [comment]
Hi,

We are investigating this

**Comment by arturmuller** [comment]
Yup, same here. Love that you can't even submit a ticket to the Vercel team. :facepalm:

Meanwhile, [Vercel Status](https://www.vercel-status.com/) is happily saying All Systems Operational.

**Comment by swarnava** [comment]
Update: we have put up status page and working on a fix: https://www.vercel-status.com/incidents/440j6wgw60n5

**Reply by ccurves** [reply]
How long is this is going to take, I have production app deployed and now just because I create a deploy to update the env variables the system is crashing. And that's another issue I've with vercel why can't the environment variables be updated automatically? I have to redeploy my app every time I make a change

**Comment by swarnava** [comment, marked as answer]
This should be fixed now. Can you try and let us know?
